### PR TITLE
According to the conditions to check whether to automatically submit pr

### LINKED ISSUE #64: [EdgeX Auto-Collector] EdgeX Auto-Collector verifying test
**What would you like to be added**:
Because the collection of edgex configuration information is automated, we need a process to verify its reliability. This program behaves like a simple e2e test. By launching this simple test we can ensure that the user is using the correct edgex configuration.
Refer to [verifying test](https://github.com/openyurtio/openyurt/blob/master/docs/proposals/20221109-edgex-auto-collector.md#automatically-audit-versions)
This issue needs to be completed after #62 and #63

**Why is this needed**:
Refer to [EdgeX Auto-colllector Proposal](https://github.com/openyurtio/openyurt/blob/master/docs/proposals/20221109-edgex-auto-collector.md#images-management)

**others**
/kind feature

### LINKED ISSUE #65: [EdgeX Auto-Collector] Set the Auto-Collector to be triggered periodically
**What would you like to be added**:
This will be the **last** issue to complete the EdgeX Auto-Collector. Set a timing trigger to enable the EdgeX Auto-Collector to periodically synchronize `edgex-compose`.

**Why is this needed**:
Refer to [EdgeX Auto-colllector Proposal](https://github.com/openyurtio/openyurt/blob/master/docs/proposals/20221109-edgex-auto-collector.md#images-management)

**others**
/kind feature

### PR DESCRIPTION
**Pull request by yangbobo6**
/kind feature
/sig iot

#### What this PR does / why we need it:
Set a timing trigger to enable the EdgeX Auto-Collector to periodically synchronize.

#### Which issue(s) this PR fixes:
Fixes #65

### CODE REVIEW

**Comment by LavenderQAQ** [comment]
Issue #64  also need to modify the e2e test.This pr only addresses issue #65.

File: .github/workflows/auto-submit-pr.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+on:
+#   schedule:
```
Comment (LavenderQAQ) [inline .github/workflows/auto-submit-pr.yaml:2]: We need to open the button of periodically launching.

File: EdgeXConfig/manifest.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+updated: "true"
+count: 13
+latestVersion: levski
+versions:
+    - main
```
Comment (LavenderQAQ) [inline EdgeXConfig/manifest.yaml:5]: We currently only have five valid versions, and the extra versions do not need to appear in the manifest.
```yaml
versions:
   - levski
   - kamakura
   - jakarta
   - ireland
   - hanoi
```

**Review by LavenderQAQ — CHANGES_REQUESTED** [review]

**Comment by LavenderQAQ** [comment]
I have a question, do we need a separate branch to run? have you tested whether starting this action on the main branch will affect the original code of the main branch? I am not sure how this action to submit pr works, will it split into a new branch to submit pr?

**Comment by yangbobo6** [comment]
> I have a question, do we need a separate branch to run? have you tested whether starting this action on the main branch will affect the original code of the main branch? I am not sure how this action to submit pr works, will it split into a new branch to submit pr?

This automatically submitted pr, as I understand it, will create a new branch. The name of this branch can be specified with a key in the yaml file, and then compare whether all the codes of the newly created branch are the same as the original branch. If they are the same, delete them. If the newly created branch is different, submit pr.

**Pull request closed without merging**